resources/web/streaming_viewer/index.html

```diff
@@ -11,18 +11,33 @@
   </head>
 
   <body>
-    <div style="padding-left:10px;height:100px">
+    <header>
       <h1>Webots streaming viewer</h1>
-      <p> Connect to: <input id="IPInput" type="text" value="localhost" /> : <input id="PortInput" type="text" value="1234" maxlength="5" size="5" /> mode:
+
+      <p> Connect to: <input id="IPInput" type="text" size="60" value="ws://localhost:1234/" />
```

Comment:
I believe the final slash is useless.
```suggestion
      <p> Connect to: <input id="IPInput" type="text" size="60" value="ws://localhost:1234" />
```